dev/docs/builtins-reference.md

```diff
@@ -251,6 +252,38 @@ w = lm(X = X, y = y)
 yp = lmpredict(X, w)
 ```
 
+## `naivebayes`-Function
+
+The `naivebayes`-function computes the conditional a-posterior probabilities of a categorical class variable given independent predictor variables using the Bayes rule.
+
+### Usage
+```r
+naivebayes(D,C,laplace,verbose)
+```
+
+### Arguments
+| Name            | Type           | Default  | Description |
+| :------         | :------------- | -------- | :---------- |
+| D               | Matrix[Double] | required | One dimensional column matrix with N rows. |
+| C               | Matrix[Double] | required | One dimensional column matrix with N rows. |
+| Laplace         | Double         | `1`      | Any Double value. |
+| Verbose         | Boolean        | `TRUE`   | Boolean value. |
+
+### Returns
+| Type           | Description |
+| :------------- | :---------- |
+| Matrix[Double] | One dimensional column matrix with N rows. |
+| Matrix[Double] | One dimensional column matrix with N rows. |
+
+### Example
+```r
```

Comment:
Can you copy paste the output of this snippet in this PR as a commit.

Tip: use the following snippet to print result matrices. 🔢 
```
print("-- printing prior --")
print(toString(prior))
print(" ")
print("--- printing class conditionals ")
print(toString(classConditionals))
```